boards/st/stm32u083c_dk/stm32u083c_dk.dts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/u0/stm32u083Xc.dtsi>
+#include <st/u0/stm32u083mctx-pinctrl.dtsi>
+#include "arduino_r3_connector.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "STMicroelectronics STM32U83C-DK board";
+	compatible = "st,stm32u083c-dk";
+
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	chosen {
+		zephyr,console = &usart2;
+		zephyr,shell-uart = &usart2;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+		green_led_1: led_1 {
+			gpios = <&gpioa 5 GPIO_ACTIVE_HIGH>;
```

Comment:
According to the datasheet on PA5 the led is blue(LD4)?
The green is on PC13 (LD3), and on PB2 there is a red led (LD5).

Reply:
I replaced the PA5 with the PC13